dashboard/app/config.go

```diff
@@ -764,6 +764,13 @@ func (cfg *Config) lastActiveReporting() int {
 	return last
 }
 
+func (cfg *Config) mainRepoBranch() (repo, branch string) {
+	if len(cfg.Repos) > 0 {
```

Comment:
This check is not necessary. We ensure there is always at least 1 repo, and code already relies on it.

Reply:
done